test/workers/backend_delete_stats_worker_test.rb

```diff
@@ -29,13 +29,13 @@ def setup
   private
 
   def assert_delete_job_params(delete_job)
+    deletejobdef = delete_job[:deletejobdef] || {}
```

Comment:
Probably better like this:
```ruby
def assert_delete_job_params(delete_job)
    [
      -> { delete_job.is_a?(Hash) && delete_job[:deletejobdef].is_a?(Hash) },
      -> { (delete_job.dig(:deletejobdef, :applications) || []).sort == applications.map(&:id).sort },
      -> { (delete_job.dig(:deletejobdef, :metrics) || []).sort == metrics.map(&:id).sort },
      -> { delete_job.dig(:deletejobdef, :users) == [] },
      -> { delete_job.dig(:deletejobdef, :from) == service.created_at.utc.to_i },
      -> { delete_job.dig(:deletejobdef, :to) == Time.now.utc.to_i }
    ].all?(&:call)
end
```

But that will be an enhancement, there is no time for this now 😄